ext/drmgr/drmgr.c

```diff
@@ -650,6 +660,7 @@ cblist_delete_local(void *drcontext, cb_list_t *l, size_t local_num)
 static void
 cblist_create_global(cb_list_t *src, cb_list_t *dst)
 {
+    memset(dst, 0, sizeof(cb_list_t));
```

Comment:
`sizeof(*dst)` (better practice in general b/c it's future-proof vs changing the type of `dst`)

Reply:
Done.